demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -587,7 +590,12 @@ export class DemoMeetingApp
     document.getElementById('button-test-sound').addEventListener('click', async e => {
       e.preventDefault();
       const audioOutput = document.getElementById('audio-output') as HTMLSelectElement;
-      const testSound = new TestSound(this.meetingEventPOSTLogger, audioOutput.value);
+      const testAudioElem = document.getElementById('test-audio') as HTMLAudioElement;
```

Comment:
… do you actually need to make this change?

Reply:
No, it is fine with just a constant AudioElement (which I had originally). I just thought to do it this way since we also had an audio element specified for the main meeting page.